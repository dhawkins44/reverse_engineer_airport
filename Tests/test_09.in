4 4 5 1000
0 2
0 0
1 1
2 1
3 1
0 1 1000
1 3 5
2 3 5