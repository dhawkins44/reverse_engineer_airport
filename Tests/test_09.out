Verify Path: 1
1010: 0 1 3 2 
